ql/src/java/org/apache/hadoop/hive/ql/io/parquet/vector/VectorizedParquetColumnReader.java

```diff
@@ -0,0 +1,13 @@
+package org.apache.hadoop.hive.ql.io.parquet.vector;
+
+import org.apache.hadoop.hive.ql.exec.vector.ColumnVector;
+import org.apache.hadoop.hive.serde2.typeinfo.TypeInfo;
+
+import java.io.IOException;
+
+public interface VectorizedParquetColumnReader {
```

Comment:
Add some comment on the `readBatch` method? Also, from the method signature it seems it should not be restricted to only Parquet. How about `VectorizedColumnReader`?